hypothesis-python/src/hypothesis/searchstrategy/strategies.py

```diff
@@ -498,14 +613,12 @@ def calc_label(self):
 
     def do_draw(self, data):
         # type: (ConjectureData) -> Ex
-        n = len(self.element_strategies)
-        assert n > 0
-        if n == 1:
-            return data.draw(self.element_strategies[0])
-
-        i = cu.integer_range(data, 0, n - 1)
-
-        return data.draw(self.element_strategies[i], label=self.branch_labels[i])
```

Comment:
This is the only place that the `branch_labels` property was used, so we can delete it and remove the relevant code from the `element_strategies` property too.  (this coverage gap is the current test failure)

Reply:
bb262f5f11a54d0c4cc5e69f97d8ddc1d7c96ee1